vendors/swedbank-pay-core-library/src/SwedbankPay/Core/Library/PaymentInfo.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+namespace SwedbankPay\Core\Library;
+
+use SwedbankPay\Core\Api\Authorization;
+use SwedbankPay\Core\Api\Response;
+use SwedbankPay\Core\Api\Transaction;
+use SwedbankPay\Core\Api\Verification;
+use SwedbankPay\Core\Exception;
+use SwedbankPay\Core\Log\LogLevel;
+
+trait PaymentInfo
+{
+    /**
+     * Do API Request
+     *
+     * @param       $method
+     * @param       $url
+     * @param array $params
+     *
+     * @return Response
+     * @throws \Exception
+     */
+    public function request($method, $url, $params = [])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method request() has a Cyclomatic Complexity of 13. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-payments/pullRequest?prid=5340032)